plugins/sql/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: sql
 title: SQL
 description: The SQL plugin allows you to run SQL queries
-version: 3.0.4
+version: 3.0.5
 supported_versions: ["MySQL 8.0.21", "MSSQL 2019 15.0.2000.5", "PostgresSQL 13.0"]
```

Comment:
Customer was not able to connect to: ```I’ve read that Azure SQL always runs on the latest version of SQL Server, which [looks like CU16 15.0.4223.1](https://support.microsoft.com/en-us/topic/kb4518398-sql-server-2019-build-versions-782ed548-1cd8-b5c3-a566-8b4f9e20293a), whereas the R7 SQL plugin documentation claims to support MSSQL 2019 15.0.2000.5, which was the RTM version of SQL Server 2019.

I couldn’t find a definitive source, and granted the versions aren’t exactly aligned, but hopefully there shouldn’t be any compatibility issues there given that they’re both 15.0.x``` Do we support the CU16 15.0.4223.1 with the update. If so can you please add the version. Also, kindly  update the Version History